Duo/duo-security/tests/test_auth_log.json

```diff
@@ -0,0 +1,47 @@
+  },
+  "expected": {
+    "message": "{\n  \"eventtype\": \"auth_log\",\n  \"access_device\": {\n    \"browser\": \"Chrome\",\n    \"browser_version\": \"67.0.3396.99\",\n    \"flash_version\": \"uninstalled\",\n    \"hostname\": null,\n    \"ip\": \"169.232.89.219\",\n    \"is_encryption_enabled\": true,\n    \"is_firewall_enabled\": true,\n    \"is_password_set\": true,\n    \"java_version\": \"uninstalled\",\n    \"location\": {\n      \"city\": \"Ann Arbor\",\n      \"country\": \"United States\",\n      \"state\": \"Michigan\"\n    },\n    \"os\": \"Mac OS X\",\n    \"os_version\": \"10.14.1\",\n    \"security_agents\": []\n  },\n  \"adaptive_trust_assessments\": {\n    \"more_secure_auth\": {\n      \"features_version\": \"3.0\",\n      \"model_version\": \"2022.07.19.001\",\n      \"policy_enabled\": false,\n      \"reason\": \"Normal level of trust; no detection of known attack pattern\",\n      \"trust_level\": \"NORMAL\"\n    },\n    \"remember_me\": {\n      \"features_version\": \"3.0\",\n      \"model_version\": \"2022.07.19.001\",\n      \"policy_enabled\": false,\n      \"reason\": \"Known Access IP\",\n      \"trust_level\": \"NORMAL\"\n    }\n  },\n  \"alias\": \"\",\n  \"application\": {\n    \"key\": \"DIY231J8BR23QK4UKBY8\",\n    \"name\": \"Microsoft Azure Active Directory\"\n  },\n  \"auth_device\": {\n    \"ip\": \"192.168.225.254\",\n    \"key\": \"DP5BJ05HI4WRBVI4Q7JF\",\n    \"location\": {\n      \"city\": \"Ann Arbor\",\n      \"country\": \"United States\",\n      \"state\": \"Michigan\"\n    },\n    \"name\": \"My iPhone X (734-555-2342)\"\n  },\n  \"email\": \"narroway@example.com\",\n  \"event_type\": \"authentication\",\n  \"factor\": \"duo_push\",\n  \"isotimestamp\": \"2020-02-13T18:56:20.351346+00:00\",\n  \"ood_software\": null,\n  \"reason\": \"user_approved\",\n  \"result\": \"success\",\n  \"timestamp\": 1581620180,\n  \"trusted_endpoint_status\": \"not trusted\",\n  \"txid\": \"340a23e3-23f3-23c1-87dc-1491a23dfdbb\",\n  \"user\": {\n    \"groups\": [\"Duo Users\", \"CorpHQ Users\"],\n    \"key\": \"DU3KC77WJ06Y5HIV7XKQ\",\n    \"name\": \"narroway@example.com\"\n  }\n}",
+    "event": {
+      "kind": "event",
+      "category": ["authentication"],
+      "type": ["info"],
+      "dataset": "auth_log"
+    },
+    "@timestamp": "2020-02-13T18:56:20.351346Z",
+    "user": {
+      "email": "narroway@example.com",
+      "id": "DU3KC77WJ06Y5HIV7XKQ"
```

Comment:
I wonder if we can fill ECS field `user.roles` with the content of `user.groups`?

Reply:
Make sense, will add it